inc/commondbrelation.class.php

```diff
@@ -637,7 +637,7 @@ function addNeededInfoToInput($input) {
 
          } else {
             // No entity link : set default values
-            $input['entities_id']  = 0;
+            $input['entities_id']  = Session::getActiveEntity();
```

Comment:
Can this value be empty, in a cron context for instance ?

In `DomainRecord` class, we have `$_SESSION['glpiactive_entity'] ?? 0`, and I guess it was done to handle creation from a cron context. @trasher Can you confirm ?